build.py

```diff
@@ -64,7 +65,13 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    '2.11.0dev': ('21.06dev', '21.04', '1.7.1', '2021.2.200', '2021.2.200')
+    '2.11.0dev':
+      ('21.06dev',   # triton container
+       '21.04',      # upstream container
+       '1.7.1',      # ORT
+       '2021.2.200', # ORT OpenVINO
+       '2021.2.200', # Standalone OpenVINO (non-windows)
+       '2021.3.0')   # Standalone OpenVINO (windows)
```

Comment:
So, it's ok to have separate OV version than ORT OpenVINO on windows? We can update the comment above that describes the problem with dlopen and how it doesn't exists in windows(or with dlls).

Still, I wonder whether shouldn't we use the identical OV version on windows as well to maintain consistency.

Reply:
Once I get windows and linux building OV then I'll work on getting all OV users to use the same version/build of OV and these 3 versions will reduce to 1 and that will make sure that we use the same version of OV for all uses.